kernel-modules/BLOCKLIST

```diff
@@ -91,3 +91,4 @@
 # https://github.com/falcosecurity/libs/issues/918
 # Since kmods are not supported anyway, block these combination
 ~.*\.el8_9\..* * mod
+~.*\.el9_4\..* ~2\.[3-9]\..* *
```

Comment:
Do we want to expand this to block RHEL 9.5+ or should we wait for it to fail?

Reply:
Hmm I'm sort of on the fence; it's _probably_ not going to compile on RHEL 9.5+ but we might want build failures to tell us one way or another when it's released

Reply:
That's fine, just wanted to double check.